### DISCUSSION TITLE
some Variant tagged by other team's members do not show up anymore in my search

### DISCUSSION THREAD

**Discussion by TOUTEE** [broadinstitute/seqr#2428] [Q&A]
**Describe the bug**
A clear and concise description of what the bug is, and what you instead expect to be happening

**Link to page(s) where bug is occurring**
https://seqr.broadinstitute.org/...

**Scope of the bug**
Are you experiencing this issue in all projects? Within specific families within one project?

**Screenshots**

**Comment by TOUTEE** [comment]
Varaints tagged by other team's member do not show up anymore in my search

**Comment by hanars** [comment]
Hi, in order for me to help you you need to provide detailed information. Please actually fill out the template with as much detail as possible. To be clear, this requires a full, detailed paragraph describing the issue with as many examples as possible, links to the pages (even if I can't access them the links are still needed), the scope of the bug (where is this happening? in every project or just some? how long has this been going on? did something change in seqr around when this stopped working?), and screenshots. I will not be able to assist you further until all of this information is provided
